md/list-of-documents.md

```diff
@@ -1,9 +1,42 @@
+
+## Exemple: Document validation process
+
+::: warning
+This tutorial assume that you are already familiar with Bonita. If not you might want to read the [Getting started](getting-started-tutorial.md) first.
+:::
+
+This exemple shows how to easily create a document validation process using Bonita Studio.  
+A document validation process is a process where a User should be able to first submit a list fo documents. Then, an other user can perform a review on the list submitted, and ask for some changes on the list. If the reviewer asks for it, then the submitter has to edit the list of documents by adding / editing / removing some documents of the list, according to the review.  
+
+Here is a preview of the process we are going to build:  
+![Validation process](images/documentValidationProcess.png)
+
+ 1. Create a process _documentValidation_, with two lanes: _Submitter lane_ and _Reviewer lane_. 
+ 2. Create also two actors: _Submitter actor_ and _Reviewer actor_. Each actor should be mapped to a different part of the organization (two different users for exemple). Set the _Submitter actor_ as initiator, and map each lane to its actor.
+ 3. Create a multiple document variable named _documents_, and a process variable of type boolean named _submissionValid_.
+ 4. Select the pool and create a contract for the data _documents_.  ⚠️ Make sure to use the button **Add from data** to create your contract, else the contract input is not bounded to the data, you will have to code many things by yourself.
+ 5. Create the instantiation form using the Studio shortcut. Rename the form into _initDocuments_, and then go back to the Studio.
+ 6. Select the task _Review_. The objective of this task is to display the documents submitted to the reviewer. He should then be able to validate or not the submission, depending on the documents. To do so, we are going to use a small trick to gain some development time.
+    - Create a contract input for the document data _documents_, using the button Add from data. This contract input will be removed in the second time, but it will allow us to generate a form which display the documents (that's the trick).
+    - Create an other contract input (using the button add this time) of type BOOLEAN, named _review_.
+    - Create the form using the Studio shortcut. Rename this form review, **and remove the file upload widget from the form** -> The form display the documents, and give the possibility to validate the submission. Save and go back to the Studio.
+    - On the task review, remove the contract input for the documents (the form has been generated, we do not need it anymore). 
+    - On the operations panel, remove the operation that update the list of documents (same idea), and add an operation to assign the value of the contract input _review_ to the process data _submitionValid_. At the end, you should have only one operation: _submitionValid_ takes value of _review_.
```

Comment:
:question: is this trick really saving time? I think the text forgtet to mention that the formOutput should also be updated to stop sending the documents...

:information_source: I think I would personnaly prefer that the tutorial explains me how to create the form without the trick. (this would be more helpful in other situations when I will need repeated containers)

Reply:
ok